src/main/resources/application.yaml

```diff
@@ -190,7 +190,8 @@ scheduling:
       cron: ${DELETE_REJECTED_FILES_CRON}
       ttl: ${DELETE_REJECTED_FILES_TTL}
     delete-complete-files:
-      enabled: false
+      enabled: ${DELETE_COMPLETE_FILES_ENABLED}
+      cron: ${DELETE_COMPLETE_FILES_CRON:false}
```

Comment:
default value false ? Also we should not need default value

Reply:
I am just restoring the build, will fix this after the build passes